### PR TITLE
Return of Stat icons

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Uses the brand spanking new ability to render clientside appearances inside of img tags within the browser to bring back the stat icons. The only server-side cost is generating refs for the atmos, there is no need to maptick appearances, and no need to generate flat icons.

Also adds support for the tgui window to be able to determine the client's BYOND version, this allows for the stat panel to change its rendering behaviour depending on the version of byond that is running on the client.

Also adds a new flag to prevent aggregation of an atom. This can be applied for the type, or for individual items.

## Why It's Good For The Game

These make it so much easier to do wire and pipe manipulation.

## Testing Photographs and Procedure

<details>
<summary>
Client Version 514
</summary>

image

</details>

<details>
<summary>
Client Version 515
</summary>

image

image

image

(Updated so that wires don't stack)

image

image

image

</details>

## Changelog
:cl:
add: Clients running 515 will now be able to see icons in the alt-click stat panel menu.
/:cl:

### CODE REVIEW

File: code/_compile_options.dm [hunk 1]
```diff
@@ -101,8 +101,8 @@
 #endif
 
 //Update this whenever the byond version is stable so people stop updating to hilariously broken versions
-#define MAX_COMPILER_VERSION 514
-#define MAX_COMPILER_BUILD 1589
+#define MAX_COMPILER_VERSION 515
+#define MAX_COMPILER_BUILD 1640
```
Comment (PowerfulBacon) [inline code/_compile_options.dm]:
```suggestion
#define MAX_COMPILER_VERSION 514
#define MAX_COMPILER_BUILD 1589
```

**Comment by PowerfulBacon** [comment]
This works with our testmerged version of 515

**Comment by EvilDragonfiend** [comment]
de-TM by merge conflict

**Comment by PowerfulBacon** [comment]
This should be fine to merge even if not running 515

**Pull request merged**